src/Kount/Util/Khash.php

```diff
@@ -52,7 +52,7 @@ public static function getSaltPhrase() {
   public static function hashPaymentToken ($token) {
     $firstSix = mb_substr($token, 0, 6, 'latin1');
     $hash = self::hash($token, 14);
```

Comment:
Yes, they are. mb_substr allows null values since php 5.4.18 and the hash function in it's current implementation will also not throw an error if it is provided with a null value. I've confirmed this through tests.